examples/layers/widgets/gestures.dart

```diff
@@ -153,22 +142,24 @@ class _GestureDemoState extends State<GestureDemo> {
   Widget build(BuildContext context) {
     return new Stack(
       children: <Widget>[
-        new GestureDetector(
```

Comment:
The better fix here is probably just to set the `fit` argument on `Stack` to `StackFit.expand`.